packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -114,13 +115,21 @@ export class GadgetConnection {
 
   enableSessionMode(options?: true | BrowserSessionAuthenticationModeOptions) {
     this.authenticationMode = AuthenticationMode.BrowserSession;
+
+    let sessionTokenStore;
```

Comment:
nit: Could we type this as `BrowserStorage` like in the class property? Doesn't need to be if it's inferred but I don't think it is

Reply:
we should need to explicitly type this here, it should be inferred from use.